### ISSUE TITLE
Does this plugin support `@use` rules

### ISSUE THREAD

**Issue by Lyliya** [csstools/stylelint-value-no-unknown-custom-properties#48]
Hey 👋 

I see that the plugin use `root.walkAtRules("import")`. Does this also support the `@use` rule as well with the alias resolver ? I cannot manage to make it work using
```scss
@use '@/components/Button/Button.tokens.css';

.class {
    border: solid var(--button-border-width); // This trigger an error, even with the variable defined in the imported file
}
```
